docs/book/v3/migration/v2-to-v3.md

```diff
@@ -3,6 +3,47 @@
 laminas-filter version 3 makes a number of changes that may affect your application.
 This document details those changes, and provides suggestions on how to update your application to work with version 3.
 
+## Signature and Behaviour Changes
+
+### AbstractFilter
+
+The deprecated method `hasPcreUnicodeSupport()` has been removed. This method had been irrelevant for a long time since reliable Unicode support in `preg_*` functions is available in all supported versions of PHP.
+
+### FilterInterface
+
+`Laminas\Filter\FilterInterface` now specifies `__invoke()` as well as `filter()` forcing all filters to also be invokable classes.
+
+In practice this is unlikely to cause problems because `AbstractFilter`, from which most filters extend already implements this method. You will however, encounter issues if you have a custom filter implementing `FilterInterface` that lacks an `__invoke()` method.
+
+Implementation is straight forward, and in most cases, adding the following method should suffice:
+
+```php
+final class MyFilter implements Laminas\Filter\FilterInterface {
+    public function filter(mixed $value): mixed
+    {
+        // Filter things…
+    }
+    
+    public function __invoke(mixed $value) : mixed
+    {
+        return $this->filter($value); 
+    }
+}
+```
+
+Also note that the signature of both `filter()` and `__invoke()` has changed to include `mixed` as both parameter and return type, therefore, it will be necessary to add any missing parameter and return types to your custom filters.
```

Comment:
```suggestion
#### Change of Parameter and Return Type

Also note that the signature of both `filter()` and `__invoke()` has changed to include `mixed` as both parameter and return type, therefore, it will be necessary to add any missing parameter and return types to your custom filters.
```